### PR TITLE
12 add support for paraspell sdk

### LINKED ISSUE #12: Add support for ParaSpell SDK
Add the ability to use ParaSpell SDK (It will be preinstalled, maybe one demonstrational component, an example for scenario can be taken from https://github.com/paraspell/ui )

### PR DESCRIPTION
**Pull request by brano-hozza**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Fix
- [x] Feature
- [x] Refactoring
- [ ] Chore

### What's new?

- [x] PR closes #12 
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR
- [ ] I've written some unit tests 🧪

### Optional

- [ ] I found edge cases

### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### CODE REVIEW

**Review by dudo50 — APPROVED** [review]

File: components/assets/AssetComponent.vue [hunk 1]
```diff
@@ -50,34 +75,71 @@ import {
...
+  )
 }
 </script>
 <style lang="scss">
-.currency-select {
-  width: 100px;
+.node-select,
+.asset-select,
+.destination-select {
+  width: 200px;
```
Comment (vikiival) [inline components/assets/AssetComponent.vue:143]: Why fixed width?

File: stores/AssetsStore.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { NODE_NAMES, TAssetDetails, TNode, TNodeAssets } from '@paraspell/sdk'
```
Comment (vikiival) [inline stores/AssetsStore.ts:1]: Rename this fille to assetStore

**Review by vikiival — CHANGES_REQUESTED** [review]
Please unify casing across the app in template files. 

Move paradspell to separated route see latest issue
